dir.props

```diff
@@ -274,6 +274,10 @@
 
   <!-- Setup properties per OSGroup -->
   <Choose>
+    <When Condition="'$(IsReferenceAssembly)' == 'true' OR'$(MSBuildProjectExtension)' == '.pkgproj'">
```

Comment:
Do you think we will have any cases where we need to set OSGroup specific properties for reference assemblies or pkgprojs? I understand why you did this but it does seem a little odd to have this condition in the OSGroup switch statement.